internal/backoff/exponential.go

```diff
@@ -34,37 +35,47 @@ type ExponentialOption func(*exponentialOptions)
 
 // exponentialOptions are the configuration options for an exponential backoff
 type exponentialOptions struct {
-	base, min, max time.Duration
-	rand           *rand.Rand
-	minMaxDiff     int64
+	first, max time.Duration
```

Comment:
nit: I don't like this style of declaring multiple variables with the same type in the same line, and we don't do it in other places in the codebase I don't think